### PR TITLE
made changes to extend datamodel with additional properties

### PR DESCRIPTION
**Pull request by marendtz**

### CODE REVIEW

File: DataServiceDCAT-AP/examples/example_extended-normalized.json
Comment (albertoabellagarcia) [inline DataServiceDCAT-AP/examples/example_extended-normalized.json:1]: The name of the file cannot be changed, so please rename example-normalized.json

File: DataServiceDCAT-AP/examples/example_extended.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (albertoabellagarcia) [inline DataServiceDCAT-AP/examples/example_extended.json:1]: name of the file has to be kept

File: DataServiceDCAT-AP/examples/example_extended.jsonld [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (albertoabellagarcia) [inline DataServiceDCAT-AP/examples/example_extended.jsonld:1]: keep the name of the file

File: DataServiceDCAT-AP/examples/example_extended.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  "id": "urn:ngsi-ld:DataServiceDCAT-AP:AgendaAnalyticsCrawling",
+  "type": "DataServiceDCAT-AP",
+  "configuration": [
+    "param1",
+    "param2"
+  ],
+  "dateCreated": "2022-06-21T08:24:35.905712+02:00",
+  "dateModified": "2022-06-22T09:24:35.905712+02:00",
+  "dataServiceDescription": "Service that crawls web documents for a specific NGSI-LD entity of the type Organization (https://smartdatamodels.org/dataModel.Organization/Organization)",
```
Comment (albertoabellagarcia) [inline DataServiceDCAT-AP/examples/example_extended.json:10]: it should be an array (change example o schema)

File: DataServiceDCAT-AP/examples/example_extended.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  "title": "AgendaAnalyticsCrawling",
```
Comment (albertoabellagarcia) [inline DataServiceDCAT-AP/examples/example_extended.json:11]: It has to be an array not a string

File: DataServiceDCAT-AP/examples/example_extended.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  "contactPoint": {
+    "name": "Contact for Support of AgendaAnalyticsCrawling service",
+    "url": "https://www.support.com"
+  },
+  "dataProvider": "urn:ngsi-ld:Organization:DataServiceCompanyXYZ",
+  "endpointURL": "https://www.crawling-service.com",
```
Comment (albertoabellagarcia) [inline DataServiceDCAT-AP/examples/example_extended.json:17]: it has to be an array not a string

**Review by albertoabellagarcia — CHANGES_REQUESTED** [review]
Change the examples according to the notes in the example.json (but the other formats have to be changed as well)

**Comment by marendtz** [comment]
> Change the examples according to the notes in the example.json (but the other formats have to be changed as well)

Do you want me to overwrite all the examples with the examples of the extended use case? Or should I adapt the existing examples with exemplary values for the added attributes? This may be complicated, because the examples use case of the EDP may not have e.g. an assetProvider.

**Comment by albertoabellagarcia** [comment]
> > Change the examples according to the notes in the example.json (but the other formats have to be changed as well)
> 
> Do you want me to overwrite all the examples with the examples of the extended use case? Or should I adapt the existing examples with exemplary values for the added attributes? This may be complicated, because the examples use case of the EDP may not have e.g. an assetProvider.

Yes you have to rewrite the examples with the extended attributes (in the 4 formats). Ideally you can just add the extended attributes to existing examples. If you want to change the values of other attributes is also possible. What you should not do is to remove attributes from the examples. 
And to check that the keyvalues examples are validated by the schema. We use this [tool](https://www.jsonschemavalidator.net/) for this validation but you can use whatever you want.

**Review by albertoabellagarcia — APPROVED** [review]

**Pull request merged**